SalesforceCommerceCloud/pwa-kit
# UI improvement for 5xx error pages

This PR improved the UI of the error page on the pwa kit based on [design](https://www.figma.com/file/gOUKasMrK16zS5qrLCTSJl/PWA-Design-Kit?node-id=2293%3A554). This is just an UI work only. The work for internationalization on this page is blocked as the error does not have access to intl object like other components in the app. This will be addressed in a different ticket after we consider to make intl accessible across the app

 **GUS**: https://gus.lightning.force.com/a07EE000005zcTkYAI
 
Since this _error page is treated as a page and it is not going to be re-used elsewhere. It makes sense to do inline styling here

## How to test-drive this PR
- `npm ci` at root if needed
- Throw an Error in `app/components/_app.jsx` inside `getProps` method to trigger an error. 
```
App.getProps = () => {
  throw new Error('this is an error')
}
```
- Navigate to `localhost:300` and you should see the error page is styled to match the design

## Changes
- /components/_error

## Todos

### General

### Backwards Compatibility

> Breaking changes include:
> * Removing a public function/Component/Prop
> * Adding required arguments to functions
> * Changing the data-types of function params or return values

### Accessibility Compliance
You must check off all items in **one** of the follow two lists:

_or..._


### Documentation


> Documentation and comments should follow the [Mobify Content Style Guide](https://docs.google.com/a/mobify.me/document/d/1jlcg5boC3MUHN7fy2n3Yu_FBGzGgTUPrnMoePzQtqE4/edit?usp=sharing) and [Mobify's Capitalization and Spelling List](https://docs.google.com/document/d/1LO7RAr2vD3LFs_bj5j0vIFMKsVFrOFfEjpCheYEH3Lg/edit?usp=sharing)
